src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -0,0 +1,333 @@
+import moment from 'moment';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  LineChart, Line, BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer,
+} from 'recharts';
+
+import Spinner from '../../components/Spinner';
+import { numberWithCommas, downloadFromGuppy } from '../dataUtils.js';
+import './PlotChart.less';
+
+
+class PlotChartAxisTick extends React.Component {
+  /* eslint-disable react/prop-types */
+  render() {
+    // type is one of ['date', 'string', 'number']
+    const { x, y, payload, axis, type } = this.props;
+
+    let formattedValue = payload.value;
+    if (type === 'date') {
+      formattedValue = `${moment(formattedValue).month() + 1}/${moment(formattedValue).date()}`;
+    } else if (type === 'number') {
+      formattedValue = numberWithCommas(formattedValue);
+    }
+    // else { // type === 'string'
+    //   const maxLength = 4;
+    //   formattedValue = formattedValue.slice(0, maxLength)
+    //     + (formattedValue.length > maxLength ? '.' : '');
+    // }
```

Comment:
remove these?

Reply:
there's a ticket to add this back and make it configurable, so i left it in